<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>830</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1069</width>
    <height>830</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1069</width>
    <height>830</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>997</x>
      <y>40</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Close</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;		
	background-color: rgb(255, 0, 0);
}
QPushButton:hover {		
	background-color: rgba(255, 0, 0, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 185, 255); background-color: none;</string>
    </property>
    <property name="text">
     <string>y(x) = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 185, 255); background-color: none;</string>
    </property>
    <property name="text">
     <string>to x =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="to_x_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color:black;

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(220, 220, 220);
background-color: rgb(98, 98, 162);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>e.g. 5*x^3 +2*x</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>801</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 185, 255); background-color: none;</string>
    </property>
    <property name="text">
     <string>From x =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_of_x">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color:black;

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="from_x">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:20px;
color:black;

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minimize">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>40</y>
      <width>17</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Minimize</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;		
	background-color: rgb(255, 170, 0);
}
QPushButton:hover {	
	background-color: rgba(255, 170, 0, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="plot_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>180</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:grey;</string>
    </property>
    <property name="text">
     <string>Plot!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 185, 255); background-color: none;</string>
    </property>
    <property name="text">
     <string>Function Plotter</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_place">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;
font-size:30px;
text-align:center;
margin:0 auto;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
